umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10.clj

```diff
@@ -137,6 +137,15 @@
     (when-let [citation (first (map :OtherCitationDetails collection-citations))]
       [:CitationForExternalPublication citation])))
 
+(defn find-first-available-distribution-price
+  "Find the first FileDistributionInformation object that contains the sub element of Fees."
+  [c]
+  (loop [dist (:FileDistributionInformation (:ArchiveAndDistributionInformation c))]
```

Comment:
nit: I believe this works fine but if you want the first non-nil element you can use `some`:
https://clojuredocs.org/clojure.core/some

So your code would simply be:
`(some :Fees (get-in c [:FileDistributionInformation :ArchiveAndDistributionInformation]))`

Reply:
I was able to get (some :Fees (get-in c [:ArchiveAndDistributionInformation :FileDistributionInformation ])) working.  Very nice. I have also updated the echo10.expected-output file.